src/main/java/uk/gov/hmcts/reform/judicialapi/configuration/LaunchDarklyConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.judicialapi.configuration;
+
+import com.launchdarkly.sdk.server.LDClient;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import uk.gov.hmcts.reform.judicialapi.util.FeatureConditionEvaluation;
+
+@Configuration
+public class LaunchDarklyConfiguration implements WebMvcConfigurer {
+
+    @Bean
+    public LDClient ldClient(@Value("${launchdarkly.sdk.key}") String sdkKey) {
+        return new LDClient(sdkKey);
+    }
+
+    @Autowired
+    private FeatureConditionEvaluation featureConditionEvaluation;
+
+    /**
+     * add here entry in registry and api endpoint path pattern like below
+     * registry.addInterceptor(featureConditionEvaluation)
```

Comment:
No LD flag for JRD API ?

Reply:
yes we created a new LD flag for this API, called `jrd-fetch-profiles`